tests-functional/pages/firstrun.py

```diff
@@ -0,0 +1,64 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.base import Base
+from pages.about_home import AboutHome
+
+
+class Firstrun(Base):
+
+    _allaboard_dialog_locator = (By.ID, 'all-aboard')
+    _dismiss_link_locator = (By.ID, 'dismiss')
+    _fxa_form_locator = (By.CLASS_NAME, 'fxaccounts')
+    _questions_one_heading_locator = (By.CSS_SELECTOR, '#all-aboard header h2')
+    _submit_button_locator = (By.CSS_SELECTOR, '#all-aboard button')
+    _values_radio_element_locator = (By.ID, 'values')
+    _what_matters_section_locator = (By.CLASS_NAME, 'what-matters')
+    _yup_radio_element_locator = (By.ID, 'yup')
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda s: self.is_element_displayed(
+            *self._allaboard_dialog_locator))
+        return self
+
+    @property
+    def question_one_heading(self):
+        return self.find_element(*self._questions_one_heading_locator).text
+
+    # completes firstrun flow, setting user on values track
+    def complete_firstrun_values_flow(self):
+        self.select_yup()
+        self.select_values()
+        self.submit_form()
+        self.click_dismiss()
+
+    def dismiss_visible(self):
+        return self.is_element_displayed(*self._dismiss_link_locator)
+
+    def fxa_form_visible(self):
+        return self.is_element_displayed(*self._fxa_form_locator)
+
+    def allaboard_dialog_visible(self):
+        return self.is_element_displayed(*self._allaboard_dialog_locator)
+
+    def submit_button_visible(self):
+        return self.is_element_displayed(*self._submit_button_locator)
+
+    def what_matters_visible(self):
```

Comment:
Add `@property` before this line (and others that don't take arguments but return a value) so that when you call it you don't need to use `()`.